query/iam/iam_subscription_owner_max_3.sql

```diff
@@ -0,0 +1,32 @@
+with owner_roles as (
+  select
+    role_name,
+    role_type, name,
+    title,
+    subscription_id
+  from
+    azure_role_definition
+  where
+    role_name = 'Owner'
+)
+select
+  -- Required columns
+  owner.subscription_id as resource,
+  case
+    when count(*) >= 3 then 'ok'
```

Comment:
It is recommended to designate up to 3 subscription owners in order to reduce the potential for breach by a compromised owner.

A maximum of 3 owners should be designated for your subscription

Reply:
changed the query to handle maximum of 3 owners per subscription.